packages/react-components/src/organisms/TeamCreateOutputExtraInformationCard.tsx

```diff
@@ -21,17 +22,20 @@ import {
 
 type TeamCreateOutputExtraInformationProps = Pick<
   ResearchOutputPostRequest,
-  'tags' | 'accessInstructions' | 'labCatalogNumber'
+  'tags' | 'accessInstructions' | 'labCatalogNumber' | 'methods'
 > & {
   tagSuggestions: NonNullable<
     ComponentProps<typeof LabeledMultiSelect>['suggestions']
   >;
   onChangeTags?: (values: string[]) => void;
   onChangeAccessInstructions?: (value: string) => void;
   onChangeLabCatalogNumber?: (value: string) => void;
+  onChangeMethods?: (value: string[]) => void;
   isSaving: boolean;
   documentType: ResearchOutputDocumentType;
   identifierRequired: boolean;
+  getResearchTags?: (type: string) => Promise<ResearchTagResponse[]>;
+  type: ResearchOutputPostRequest['type'] | '';
```

Comment:
Is this typing correct, ResearchOutputIdentifierType? Or is this a different type?

Reply:
This might be empty string when you creating new RO as the type is not yet selected